Python/Product/ExternalProfilerDriver/DWJsonModel.cs

```diff
@@ -0,0 +1,165 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Microsoft.PythonTools.Profiling.ExternalProfilerDriver
+{
+    // Most numerical integer data is represented in dwjson as a 64-bit unsigned integer
+    // Due to JSON limitations, this is represented as a 2D named tuple, where the fields are
+    // "l" (low) and "h" (high), each containing a 32-bit *signed* integer.
+    // In this way, the max value (0xffffffffffffffff) is represented as {"h" : -1, "l" : -1}
+    public class LongInt
```

Comment:
Avoid making anything public unless it needs to be. As this is an executable, in theory nothing should be public. (Doesn't have to be explicitly private - simply using `class LongInt` is fine)